src/primitive_esp32.cc

```diff
@@ -310,6 +311,23 @@ PRIMITIVE(enable_external_wakeup) {
   return process->program()->null_object();
 }
 
+PRIMITIVE(enable_touchpad_wakeup) {
+#ifndef CONFIG_IDF_TARGET_ESP32C3
+  esp_err_t err = esp_sleep_enable_touchpad_wakeup();
+  if (err != ESP_OK) {
+    ESP_LOGE("Toit", "Failed: sleep_enable_touchpad_wakeup");
+    OTHER_ERROR;
+  }
+  err = esp_sleep_pd_config(ESP_PD_DOMAIN_RTC_PERIPH, ESP_PD_OPTION_ON);
+  if (err != ESP_OK) {
+    ESP_LOGE("Toit", "Failed: sleep_enable_touchpad_wakeup - power domain");
+    OTHER_ERROR;
+  }
+  keep_touch_active();
```

Comment:
There is no way to disable this again?

Reply:
Not at the moment. Once you enabled wakeup from deepsleep with touch-pins you can't disable it again.
We could add a function to `esp32` to disable wakeups again.
We don't have it for gpio either.